kvmagent/kvmagent/test/ha_testsuite/helper.py

```
@@ -1,9 +1,124 @@
+    set_max_sectors_kb=0
+    corrupt_result=0
+    acquire_last_result=1
+    renewal_last_result=1
+    acquire_last_attempt=4189
+    acquire_last_success=4189
+    renewal_last_attempt=427758
+    renewal_last_success=427758
+'''
+    # mock return value of get_sanlock_status
+    sanlock.get_sanlock_status = mock.Mock(
+        return_value=SanlockClientStatusParser(ret))
```

Ruff (0.8.2)

Undefined name `SanlockClientStatusParser`

(F821)